docs/source/io_formats/nuclear_data.rst

```diff
@@ -0,0 +1,353 @@
+.. _usersguide_nuclear_data:
+
+========================
+Nuclear Data File Format
+========================
+
+---------------------
+Incident Neutron Data
+---------------------
+
+
+**/<nuclide name>/**
+
+:Attributes: - **Z** (*int*) -- Atomic number
+             - **A** (*int*) -- Mass number
+             - **metastable** (*int*) -- Metastable state
```

Comment:
Add something like `Zero indicates the ground state; one indicates the first excited state; and so on` just to be explicit